src/backend/access/transam/parallel.c

```diff
@@ -77,6 +77,10 @@
 #define PARALLEL_KEY_RELMAPPER_STATE		UINT64CONST(0xFFFFFFFFFFFF000D)
 #define PARALLEL_KEY_UNCOMMITTEDENUMS		UINT64CONST(0xFFFFFFFFFFFF000E)
 
+/* Hooks for communicating babelfish related information to parallel worker */
+babelfixedparallelstate_insert_hook_type babelfixedparallelstate_insert_hook;
+babelfixedparallelstate_restore_hook_type babelfixedparallelstate_restore_hook;
+
```

Comment:
Hook name should be generic, does not need to have babelfish in the name. Also it should be based on the original function ie, InitializeParallelDSM.

Reply:
Updated in latest rev.